src/elf.rs

```diff
@@ -1124,16 +1124,25 @@ pub struct Rela64<E: Endian> {
 }
 
 impl<E: Endian> Rela64<E> {
+    pub(crate) fn get_r_info(&self, endian: E, is_mips64el: bool) -> u64 {
+        let mut t = self.r_info.get(endian);
+        if is_mips64el {
+            t = (t << 32) | ((t >> 8) & 0xff000000) | ((t >> 24) & 0x00ff0000) |
+                ((t >> 40) & 0x0000ff00) | ((t >> 56) & 0x000000ff);
```

Comment:
What is this doing?

Reply:
On this arch the byte order of the field is 56784321.

Reply:
Huh didn't know that. So someone decided to use individual fields instead of the u64 for this arch, and now everyone else is left with this baggage...

LLVM does the same thing as this, so looks fine to me.